src/Microsoft.AspNet.Mvc.DataAnnotations/AttributeAdapterBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Microsoft.AspNet.Mvc.ModelBinding.Validation;
+using Microsoft.Extensions.Localization;
+
+namespace Microsoft.AspNet.Mvc.DataAnnotations
+{
+    public interface IAttributeAdapter : IClientModelValidator
```

Comment:
Why not just use the abstract class below though? What's the value of the interface?